zerver/tests/test_import_export.py

```diff
@@ -965,6 +965,11 @@ def assert_realm_values(f: Callable[[Realm], object]) -> None:
             imported_realm.authentication_methods_dict(),
         )
 
+        self.assertEqual(
+            Message.objects.filter(realm=original_realm).count(),
+            Message.objects.filter(realm=imported_realm).count(),
+        )
```

Comment:
This is a nice test detail to do. Should we have something similar for the slack import tests?

Reply:
I'm struggling to find a good way of nicely grabbing the amount of messages from Slack data (someone more familiar with the data format may be able to though...), so I went with hard-coding which should be good enough because it should only change if the fixtures change i think:
```
self.assertEqual(Message.objects.filter(realm=realm).count(), 82)
```

I've confirmed `82` is also the number of imported messages before my commits happen, to verify correctness of the number.

Reply:
Sounds good!